gradle/dependencies.gradle

```diff
@@ -142,7 +142,7 @@ libs += [
   apachedsMavibotPartition: "org.apache.directory.server:apacheds-mavibot-partition:$versions.apacheds",
   apachedsJdbmPartition: "org.apache.directory.server:apacheds-jdbm-partition:$versions.apacheds",
   argparse4j: "net.sourceforge.argparse4j:argparse4j:$versions.argparse4j",
-  bcpkix: "org.bouncycastle:bcpkix-jdk15on:$versions.bcpkix",
+  bcpkix: "org.bouncycastle:bcpkix-jdk18on:$versions.bcpkix",
```

Comment:
This changes the version of bouncycaste which is compatible from JDK 1.8 onwards instead of earlier JDK 1.5.